admin/metrics.md

```diff
@@ -7,3 +7,119 @@ Metrics allow you to track the number of people using Coder on a day-to-day
 basis. Access the following view from **Manage > Metrics**.
 
 ![Metrics UI](../assets/admin/metrics.png)
+
+## Filters
+
+You can filter the report by the following properties:
+
+- **Organization**: Only show user activity within a specific
+  [organization](./organizations/)
+
+- **Action**: Only show users who performed a specific action within the time
+  period
+
+  - **Any**: Any of the below actions count towards activity
+
+  - **IDE**: Used a web IDE in a workspace (Code Web, JetBrains, PyCharm,
+    Jupyter, RStudio)
+
+  - **Login**: Logged in to Coder
+
+  - **App**: Used a custom
+    [workspace application](../workspaces/applications.md) in a workspace
```

Comment:
I wonder if we should somehow mention it only tracks long lived requests such as websockets. Short lived connections will appear only as 1 "event"